synapse/telepath.py

```diff
@@ -1281,7 +1292,7 @@ async def _teleLinkLoop(self):
                 now = time.monotonic()
                 if now > lastlog + 60.0:  # don't logspam the disconnect message more than 1/min
                     url = s_urlhelp.sanitizeUrl(zipurl(urlinfo))
-                    logger.exception(f'telepath client ({url}) encountered an error: {e}')
+                    logger.warning(f'telepath client ({url}) encountered an error: {e}')
```

Comment:
If intent is to change this from an `error` to a `warning` we should still include `exc_info=True` so that we log the actual exception information as well.